src/c++/vdo/user/vdoStatsWriter.c

```diff
@@ -0,0 +1,1444 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright 2023 Red Hat
+ *
+ * If you add new statistics, be sure to update the following files:
+ *
+ * ../base/statistics.h
+ * ../base/message-stats.c
+ * ../base/pool-sysfs-stats.c
+ * ./messageStatsReader.c
+ * ../../../perl/Permabit/Statistics/Definitions.pm
+ */
+
+
```

Comment:
extra blank line at 13-14.

Reply:
I have updated the file and removed the blank line.  Thanks.